openstack/containerinfra/v1/certificates/testing/fixtures.go

```diff
@@ -28,10 +28,37 @@ const CertificateResponse = `
 	]
 }`
 
+const CreateCertificateResponse = `
```

Comment:
Here's the response I'm seeing:

```json
{
  "bay_uuid": "7341c88d-027d-46fe-ac2e-5a0cc0ee8a73",
  "cluster_uuid": "7341c88d-027d-46fe-ac2e-5a0cc0ee8a73",
  "csr": "FOO",
  "links": [
    {
      "href": "http://example.com:9511/v1/certificates/7341c88d-027d-46fe-ac2e-5a0cc0ee8a73",
      "rel": "self"
    },
    {
      "href": "http://example.com:9511/certificates/7341c88d-027d-46fe-ac2e-5a0cc0ee8a73",
      "rel": "bookmark"
    }
  ],
  "pem": "BAR"
}
```

`cluster_uuid` is returned, but I might be testing with a newer version of Magnum.

One thing that might be helpful is to set `OS_DEBUG=1` when running the acceptance tests to see the API traffic:

```shell
$ OS_DEBUG=1 go test -v -tags "fixtures acceptance" -run TestCertificatesCRUD github.com/gophercloud/gophercloud/acceptance/openstack/containerinfra/v1 -count=1
```

Usually I write the acceptance test before the unit test so I can use the acceptance test to generate the fixtures.

Reply:
> Support both bay_uuid and cluster_uuid in gophercloud?

This is the safest option and is probably the best way to go for the time being. At least then we have initial support for both and can drop the old `bay`-style in the future.